### ISSUE TITLE
Reverted and Merged a Branch, Need Original Branch Merged Instead

### ISSUE THREAD

**Issue by powellmad** [nss-day-cohort-46/How-to-Ask-for-Help#36]
**_Ticket MUST be completed before getting help from an instructor_**
#### Name:
Madison Powell
#### Group:
Palpatine Amandine
#### Book / Chapter: 
Sprinkles of Joy
#### Have you searched the other issue tickets? If not, do that first. 
- [x] Yes
- If No, stop and search.

####  REQUIRED Add Tags to Issue Ticket and Assign to Project Board
 

- [x] Added label to specific project/assignment that you need help with
- [x] Added to `c46 Help-Thread` project board

#### REQUIRED Describe your issue:
>> I merged a branch, reverted it because I thought it was approved but wasn't, and merged it. Then, I realized that reverting a branch removes the recent changes, so I tried to merge the original branch again. I get a merge conflict notification, but have no prompts to resolve them in vscode. 

#### REQUIRED What have you googled? **_(You must have at least 3 links related to your issue)_**

- [x] 1. https://stackoverflow.com/questions/38216541/visual-studio-code-how-to-resolve-merge-conflicts-with-git/44682439
- [x] 2. https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reverting-a-pull-request
- [x] 3. https://stackoverflow.com/questions/42860234/how-to-undo-a-merge-in-github
- [x] 4. https://githowto.com/staging_changes

#### REQUIRED: What have you tried? Tell us everything you have tried. 
>> The original branch was still active and had the desired code save in the branch. I committed and pushed again to create a new pull request. Because there were no changes to commit, I added some comments to the code, then tried again. VSCode is just not prompting to fix merge conflicts. I also checked for staged changes. There weren't any the first time, but after adding the comments in, there are many staged changes? I looked up git reset, but I'm not clear on what that does.

##### Conflict Page:
https://github.com/nss-day-cohort-46/sprinkles-of-joy-palpatine-amandine/pull/25/conflicts
##### Code Location:
https://github.com/nss-day-cohort-46/sprinkles-of-joy-palpatine-amandine/blob/396cc300a8e9a6d91d9fba3588ee7d3cf87e0968/scripts/orders/OpenCart.js#L71

**Comment by powellmad** [comment]
You just have to make sure there are no unsaved changes/tabs in vscode so the merge conflicts will be prompted.
